elliottlib/util.py

```diff
@@ -687,3 +687,8 @@ def go_suffix_for_arch(arch: str) -> str:
 def brew_suffix_for_arch(arch: str) -> str:
     arch = brew_arch_for_go_arch(arch)  # translate either incoming arch style
     return brew_arch_suffixes[brew_arches.index(arch)]
+
+
+def chunk(l, n):
```

Comment:
How about adding type hints to this function?

Reply:
I would also pick more meaningful names for parameters: as I understand, "l" will always be a bug list while "n" is the batch size

Reply:
Good point, added